axonserver/src/main/java/io/axoniq/axonserver/grpc/GrpcFlowControlledDispatcherListener.java

```diff
@@ -38,25 +41,34 @@
     private final Future<?>[] futures;
     private volatile boolean running = true;
 
-    public GrpcFlowControlledDispatcherListener(FlowControlQueues<T> queues, String queueName, StreamObserver<I> inboundStream, int threads) {
+    public GrpcFlowControlledDispatcherListener(FlowControlQueues<T> queues, String queueName,
+                                                StreamObserver<I> inboundStream, int threads) {
         this.queues = queues;
         this.queueName = queueName;
         this.inboundStream = inboundStream;
         futures = new Future[threads];
     }
 
     private void process() {
+        getLogger().debug("Starting listener for {} ", queueName);
+        while (running && permitsLeft.get() > 0) {
+            try {
+                processNext();
+            } catch (Exception e) {
+                getLogger().warn("Problem occurred while processing the next message", e);
```

Comment:
Do we want to return the fact that sending the message failed to the caller? Otherwise the caller would just get a timeout message.

Reply:
IMO we can leave this improvement for another PR.